src/phoenix/server/api_key_validator.py

```diff
@@ -0,0 +1,65 @@
+from asyncio import create_task, sleep
+from typing import Any, Dict, cast
+
+import jwt
+from jwt import InvalidTokenError
+from sqlalchemy import and_, select
+
+from phoenix.auth import JwtPayload
+from phoenix.config import PHOENIX_SECRET
+from phoenix.db import models
+from phoenix.server.types import DaemonTask, DbSessionFactory
+
+
+class ApiKeyValidator(DaemonTask):
+    def __init__(
+        self,
+        db: DbSessionFactory,
+        sleep_seconds: int = 1,
```

Comment:
```suggestion
        sleep_seconds: int = 10,
```

Probably okay if we're less aggressive on the refresh.

Reply:
probably fine performance-wise, and it makes the integration test a little easier (because it's a subprocess)